container-engine-lib/lib/backend_impls/docker/docker_kurtosis_backend/engine_functions/create_engine.go

```diff
@@ -27,6 +30,11 @@ const (
 	maxWaitForEngineAvailabilityRetries         = 10
 	timeBetweenWaitForEngineAvailabilityRetries = 1 * time.Second
 	logsStorageDirpath                          = "/var/log/kurtosis/"
+	removeLogsWaitHours                         = 6
+)
+
+var (
+	logRemovalTicker = time.NewTicker(removeLogsWaitHours * time.Hour)
```

Comment:
do you need that to be a static variable here? Can't you create it where you need it?